infra/aws/terraform/prow-build-cluster/resources/monitoring/node-exporter/cluster-role.yaml

```diff
@@ -0,0 +1,15 @@
+kind: ClusterRole
+apiVersion: rbac.authorization.k8s.io/v1
+metadata:
+  name: psp-node-exporter
+  labels:
+    app: node-exporter
+rules:
+  - apiGroups:
+      - extensions
+    resources:
+      - podsecuritypolicies
```

Comment:
Same as before: PSPs are removed in Kubernetes 1.25, we perhaps don't need this file.